templates/contact.html

```diff
@@ -8,40 +8,43 @@
 <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"></script>
 {% endblock head %}
 
-{% macro row(title, slug, email, people) %}
-<tr>
-  <td> <!--class="d-flex align-items-center gap-2">-->
-    <div class="fs-4 icon baseline"><img src="{{ get_url(path="assets/exec-icons/" ~ slug ~ ".svg") }}"></div>
-    {{ title }}
-  </td>
-  <td>
-    {{ people | join(sep=", ") }}
-  </td>
-  <td>
-      <a target="_blank" rel="noopener" href="mailto:{{ email }}"
-        aria-label="Email our {{ title }}">{{ email }}</a>
-  </td>
-</tr>
-{% endmacro %}
-
 {% block content %}
 
 {{ resource.content | safe}}
 
-<table class="table table-striped exec-table">
-  <thead>
-    <tr>
-      <th>Role</th>
-      <th>People</th>
-      <th>Contact</th>
-    </tr>
-  </thead>
-  <tbody>
-      {{ self::row(title="Exec", slug="all", email="exec@uwcs.co.uk", people=["All Exec"]) }}
-    {% for role in section.pages %}
-      {{ self::row(title=role.title, slug=role.slug, email=role.extra.email, people=role.extra.people) }}
-    {% endfor %}
-  </tbody>
-</table>
+{% for role in section.pages %}
+  <div class="role">
+    <div class="fs-4 icon baseline exec-title">
+      <img src="{{ get_url(path='assets/exec-icons/' ~ role.slug ~ '.svg') }}" alt="{{ role.title }}">
+      {% if role.extra.people | length > 1 %}
+        <h3>{{ role.extra.plural }}<h3>
+      {% else %}
+        <h3>{{ role.title }}<h3>
+      {% endif %}
+    </div>
+    <a href="mailto:{{role.extra.email}}" aria-label="Email our {{role.title}}">{{role.extra.email}}</a>
+  </div>
+{% endfor %}
+<div class="exec-cards">
+{% set harmony = true %}
+{% for role in section.pages %}
+  {% for person in role.extra.people %}
+    {# because Harmony just has to be special they have 2 roles #}
```

Comment:
Perhaps define a list of duplicate people in a extra field on the exec list, something like
```
[extra]
multiple_roles = {"Harmony Bouabid" = "Publicity  & Inclusivity Officer"}
```
Then in here, when one of these is matched,read the role from the dict, and build a list of multi-role people who have been seen and avoid inserting them a 2nd time